### PR TITLE
feat(core): add graph workflow group and remove subcommand

### LINKED ISSUE #2081: Add renku workflow remove command
Add a `renku workflow remove` command as outlined in the RFC

https://github.com/SwissDataScienceCenter/renku-python/blob/rfc-001-workflow-ux/design/001-new-workflow-commands/001-new-workflow-commands.md#renku-workflow-remove

### PR DESCRIPTION
**Pull request by vigsterkr**
# Description

Fixes #2081

### CODE REVIEW

**Comment by vigsterkr** [comment]
had to restart workflows as one failed due to some VM connection issues (unrelated to renku)

**Review by Panaetius — CHANGES_REQUESTED** [review]
Nice work. I think we should also check on `renku run` that a name is not already in use (i.e. a plan with that name that isn't invalidated exists).

File: renku/core/incubation/graph.py [hunk 1]
```diff
@@ -453,3 +457,50 @@ def _add_to_dataset(
...
+
+def remove_workflow():
+    """Return a command for removing workflow."""
+    command = Command().command(_remove_workflow).lock_project()
+    return command.require_migration().with_commit(commit_only=GRAPH_METADATA_PATHS)
+
+
+def _remove_workflow(client, name: str, force: bool):
+    """Remove the given workflow."""
+    if not force:
```
[the remark this answers is not in the document]
Comment (vigsterkr) [reply renku/core/incubation/graph.py:470]: yeah i was contemplating with this option as well. 👍

**Review by Panaetius — APPROVED** [review]
Looks great now, thank you!

**Pull request merged**